.travis.yml

```diff
@@ -0,0 +1,7 @@
+os:
+  - linux
+services:
+  - docker
+script:
+  - docker build -t test .
+  - docker run -e TRELLO_KEY  -e TRELLO_TOKEN -e TRELLO_BOARD -e SECRET_KEY -e FLASK_APP='todo_app/app' -e FLASK_ENV='development' -e FLASK_SKIP_DOTENV='True' test
```

Comment:
- Env vars like `FLASK_APP='todo_app/app'` could just be set in the Dockerfile as you're never going to want to pass in something different.
- Your e2e test should not need a TRELLO_BOARD env var - it should be creating its own temporary board.
- You are running all of your tests at the same time. It would be nicer to run only the e2e tests with real credentials to minimise risk of using real data. Put the e2e test file in its own folder and then you can run:
```
docker run test todo_app/tests
docker run -e ... -e ... -e ... todo_app/e2e_tests
```
Plus you're actually implicitly relying on the order of your tests currently. Your e2e test runs first and then your integration test loads .env.test. If the order were reversed (e.g. if you rename the files), then the integration test would load .env.test first, clobbering the real env vars and the e2e test would fail.